website/docs/components/form/input-group/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,114 @@
+## Usage
+
+An Input Group is intentionally published as an agnostic container element that wraps one or more inputs and actions, making the relevant use cases broad.
+
+!!! Info
+
+The way in which these components are published and assembled together are fundamenally different between the Figma Ember components.
+
+- In Figma, multiple primitive components are published (`ButtonType` and `InputType`) that are intended to be assembled in an auto layout container. A pre-assembled `Base` component is also available for more simple instances.
+- The Ember component acts as a container to enforce consistent styling and grouping of nested Helios form components. It does not handle any logic or functionality out of the box, this is left up to the nested form elements.
+!!!
+
+### When to use
+
+- When filtering complex data, searching within a set of filters, and creating complex filtering patterns.
+- When collecting information in a form that benefits from a "connected" visual representation, e.g., a key/value pair.
+- When an action performs a generative function rather than a submit function, e.g., generating API credentials.
+- As a complex data input, e.g., uploading a document or connecting to an external resource.
```

Comment:
That's a bit of the rub with this component, visually an upload could use the primitives defined in Figma, but the component in Ember would use a different button/input type (check me on this @alex-ju, not 100% sure on the exact implementation). I'll remove this point for now though.

Reply:
Sorry for the later reply on this – it slipped through. Indeed, I see that in Figma it can be achieved using this component but in code, we have a semantic element that generates both the button and the 'field' and we'd like want to make use of it, for usability and accessibility considerations (can expand on this).